Automattic/jetpack
# Jetpack Boost: Add livereload

This adds livereload to reload CSS when styles change to Jetpack Boost when the command `npm run devlive` is run.

To enable livereloading, the Livereload extension has to be installed:

https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en


### Other information:

## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
1. Apply patch
2. Install the extension
3. run `npm run devlive`
4. Navigate to your boost test site
5. Enable livereload extension
6. Change a style and observe the live changes after a second or two.
